lib/screens/tela3.dart

```diff
@@ -0,0 +1,272 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+
+
+
+// Create a corresponding State class.
+// This class holds data related to the form.
+class MyCustomFormState extends State<MyCustomForm> {
+  // Create a global key that uniquely identifies the Form widget
+  // and allows validation of the form.
+  //
+  // Note: This is a GlobalKey<FormState>,
+  // not a GlobalKey<MyCustomFormState>.
+  final _formKey = GlobalKey<FormState>();
+  final myFocusNode = new FocusNode();
+
+  @override
+  Widget build(BuildContext context) {
+    // Build a Form widget using the _formKey created above.
+    return Form(
+
+      key: _formKey,
+      child: Column(
+
+        children: <Widget>[
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Container(
+              child: Padding(
+                padding: const EdgeInsets.fromLTRB(33.0, 108.0, 0.0, 1.0),
+                child: Text(
+                    'Responda!',
+                    style: TextStyle(
+                        color: Colors.black,
+                        fontWeight: FontWeight.w700,
+                        fontFamily: 'DM Sans',
+                        fontStyle: FontStyle.normal,
+                        fontSize: 24)
+                ),
+              ),
+            ),
+          ),
+
+
+
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Container(
+              child: Padding(
+                padding: const EdgeInsets.fromLTRB(33.0, 28.0, 33.0, 2.0),
+                child: Text(
+                  'Responda o quiz com base nos TERMOS para obtrer acesso!',
+                  maxLines: 3,
+                  overflow: TextOverflow.ellipsis,
+                  style: TextStyle(
+                      color: Colors.black,
+                      fontWeight: FontWeight.w500,
+                      fontStyle: FontStyle.normal,
+                      fontFamily: 'DM Sans',
+                      letterSpacing: 0.4,
+                      fontSize: 14),
+                ),
+              ),
+            ),
+          ),
+
+
+          Align(
```

Comment:
os campos do formulário tu pode tentar componentizar, criar um arquivo dentro da pasta /screens/core chamado talvez de text_form_field onde tu passa como parâmetro o label, o hint da pergunta e o validador, só que o validator tu passaria como uma função na branch feature/chat-screen se for na pasta lib/screens/core/circle_button.dart tem alguns exemplos onde uso o voidcallback, o objetivo é permitir que outras telas consigam reutilizar esse mesmo TextFormField que tu criou passando o conteúdo adequado para a tela deles.

outra coisa o TextFormField tem uns atributos como onchange que poderias implementar já também, nesse caso tu também usaria o voidcallback passando parâmetro o valor que esse atributo envia(veja documentação para entender melhor)